src/routes/AccountRoute/AccountRouteStyled.js

```diff
@@ -67,12 +78,17 @@ height: 64px;
 }
 `;
 
-export const ChangeAvatar = styled.a`
+export const ChangeAvatar = styled(BasicButton)`
 padding-top: 8px;
 color: var(--color-brand-blue);
+width: 120px;
+
+@media (max-width: 767px) {
+}
```

Comment:
If you don't need this mediaQuery let's remove it

Reply:
need i.